sumologic/resource_sumologic_token_test.go

```diff
@@ -0,0 +1,194 @@
+package sumologic
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+)
+
+func TestAccSumologicToken_basic(t *testing.T) {
+	var token Token
+
+	testName := "Test Terraform Token"
+	testDescription := "Description tf test token"
+	testStatus := "Active"
```

Comment:
maybe we can use a function similar to `getRandomParams()` which is present in other test files, to set the values of these variables in all the tests.

Reply:
@vsinghal13 I found the usage of method `getRandomizedParams()` in `resource_sumologic_cloud_to_cloud_source_test.go`. However, I didn't find its definition. How is this method working right now?

Reply:
https://github.com/SumoLogic/terraform-provider-sumologic/blob/master/sumologic/resource_sumologic_s3_source_test.go#L14